ExecuteLatencyHistogramAdviceTest occasionally failed

## Bug Report

### Which version of ShardingSphere did you use?

master - 06d06233fa7c47afa47858a7f193ab1b000eb810

This issue may be caused by https://github.com/apache/shardingsphere/pull/23484

### Expected behavior

Unit tests passed stably.

### Actual behavior

https://github.com/apache/shardingsphere/actions/runs/3890302698/jobs/6639304846#step:5:18097

```
[INFO] Running org.apache.shardingsphere.agent.plugin.metrics.core.advice.proxy.ExecuteLatencyHistogramAdviceTest
Error:  Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 0.538 s <<< FAILURE! - in org.apache.shardingsphere.agent.plugin.metrics.core.advice.proxy.ExecuteLatencyHistogramAdviceTest
Error:  org.apache.shardingsphere.agent.plugin.metrics.core.advice.proxy.ExecuteLatencyHistogramAdviceTest.assertExecuteLatencyHistogram  Time elapsed: 0.524 s  <<< FAILURE!
java.lang.AssertionError: 

Expected: a value greater than <500.0>
     but: <500.0> was equal to <500.0>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:8)
	at org.apache.shardingsphere.agent.plugin.metrics.core.advice.proxy.ExecuteLatencyHistogramAdviceTest.assertExecuteLatencyHistogram(ExecuteLatencyHistogramAdviceTest.java:44)
```

--- a/agent/plugins/metrics/core/src/test/java/org/apache/shardingsphere/agent/plugin/metrics/core/advice/proxy/ExecuteLatencyHistogramAdviceTest.java
+++ b/agent/plugins/metrics/core/src/test/java/org/apache/shardingsphere/agent/plugin/metrics/core/advice/proxy/ExecuteLatencyHistogramAdviceTest.java
@@ -27,7 +27,7 @@ import org.junit.Test;
 import java.lang.reflect.Method;
 
 import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.greaterThan;
+import static org.hamcrest.Matchers.greaterThanOrEqualTo;
 import static org.junit.Assert.assertTrue;
 import static org.mockito.Mockito.mock;
 
@@ -41,6 +41,6 @@ public final class ExecuteLatencyHistogramAdviceTest extends MetricsAdviceBaseTe
         Thread.sleep(500L);
         advice.afterMethod(targetObject, mock(Method.class), new Object[]{}, null);
         assertTrue(MetricsPool.get(MetricIds.PROXY_EXECUTE_LATENCY_MILLIS).isPresent());
-        assertThat(((FixtureWrapper) MetricsPool.get(MetricIds.PROXY_EXECUTE_LATENCY_MILLIS).get()).getFixtureValue(), greaterThan(500D));
+        assertThat(((FixtureWrapper) MetricsPool.get(MetricIds.PROXY_EXECUTE_LATENCY_MILLIS).get()).getFixtureValue(), greaterThanOrEqualTo(500D));
     }
 }

